server/schedulers/balance_leader_test.go

```diff
@@ -36,3 +41,30 @@ func TestBalanceLeaderSchedulerConfigClone(t *testing.T) {
 	conf2.Ranges[1] = keyRanges2[1]
 	re.NotEqual(conf.Ranges, conf2.Ranges)
 }
+
+func BenchmarkCandidateStores(b *testing.B) {
```

Comment:
The `BenchmarkCandidateStores` here is not a real benchmark I think, it will count the setup process also. We only need to bench `binarySearchStores`.

Reply:
need to use `ResetTimer` and `b.N`?